test/e2e/suites/import-gitops-v3/import_gitops_v3_test.go

```diff
@@ -215,11 +213,6 @@ var _ = Describe("[Azure] [RKE2] - [management.cattle.io/v3] Create and delete C
 			Namespace:    topologyNamespace,
```

Comment:
```suggestion
			TargetNamespace:    topologyNamespace,
```

Since it has to be applied to local cluster and used from CC namespace.